source/host/KTestSystem.cc

```diff
@@ -188,6 +189,23 @@ void KTestSystem::parseOptions(int argc, char **argv) {
             ASSERT(i + 1 < argc);
             krkgPath = argv[++i];
 
+            break;
+        case Host::EOption::Framecount:
+            ASSERT(i + 1 < argc);
+            {
+                int temp_frame = 0;
+                if (strlen(argv[++i]) > 5) {
+                    PANIC("Target Frame has too many digits");
+                }
+                temp_frame = atoi(argv[i]);
+                if (temp_frame < 0 || temp_frame > std::numeric_limits<u16>::max()) {
+                    PANIC("Target Frame is out of bounds (expected 0, 65535), got %d\n",
+                            temp_frame);
+                }
+
+                frameCount = temp_frame;
```

Comment:
Since you're just going to panic if the value is ever unsafe, I would just get rid of `temp_frame` entirely and just assign directly to frameCount